src/components/CashCrop/index.jsx

```diff
@@ -154,17 +130,20 @@ const CashCrop = ({ barebone = false }) => {
               }}
               mt={6}
             >
-              <NavButton onClick={() => navigate('/covercrop')}>BACK</NavButton>
               <NavButton
-                onClick={() => navigate('/output')}
-                disabled={
```

Comment:
why did this disabled logic get removed?

Reply:
They are not removed, it's still in the Next button's onClick. This line is aligned to the Back button